CHANGELOG.mkd

```diff
@@ -3,6 +3,7 @@ CHANGELOG
 
 Unreleased
 ----------
+- Use %LOCALAPPDATA% for Windows cachedir [RK-347](https://tickets.puppetlabs.com/browse/RK-347)
```

Comment:
```suggestion
- (RK-347) Use %LOCALAPPDATA% for Windows cachedir [#1049](https://github.com/puppetlabs/r10k/issues/1049)
```

Reply:
Seems weird to call it RK-347 without a direct link to what that means; Given the precedent of preceding comments in the changelog, it makes me think the format specified in the autogenerated GitHub comment is wrong, especially since we are focusing on using GitHub issues? This is also extra weird since we reopened the ticket, so... 🤷 

![children are wrong](https://i.kym-cdn.com/photos/images/facebook/000/645/713/888.jpg)

Reply:
The autogenerated comment was intended to capture the fact that we're primarily using GitHub issues, but should still reference JIRA tickets if they exist (in my opinion). It matches how we've done things in the past for both JIRA and GH. But we can talk about changing it if we want to do something else?

Since we've said we're using GH issues and there is an issue identical to the ticket, I think it makes sense to reference that.

Reply:
I think it's incorrect to lead with the old RK ticket number; putting it first gives it too much emphasis on it and makes it harder to skim the changelog quickly. There's also no indication that the ticket number is optional, which will lead to even more confusion for authors unaware of the RK project.

I'm fine with switching over to the GH issue.

Reply:
We have lead with RK ticket numbers in the past.. and we haven't totally completed the transition to GH-only. 
Curious if others have opinions on this.

Reply:
I think this should only reference the r10k issue and put it in the trailing position like the others. I reopened the ticket so we could track it in our sprint, but I think in general we want stuff like that to also have associated issues that can be browsed in this repo, to keep things consistent for the community. Those issues are what should be referenced here.